runtime/dart_isolate.h

```diff
@@ -430,7 +433,8 @@ class DartIsolate : public UIDartState {
               fml::WeakPtr<ImageDecoder> image_decoder,
               std::string advisory_script_uri,
               std::string advisory_script_entrypoint,
-              bool is_root_isolate);
+              bool is_root_isolate,
+              std::shared_ptr<VolatilePathTracker> volatile_path_tracker_);
```

Comment:
Remove the trailing underscore on `volatile_path_tracker_`

Reply:
Oops.